holidayextras/brands
# OPS-2542 BCP brand on tripapp

#### What does this PR do? (please provide any background)
Making BCP branded correctly on tripapp. Also makes changes so modals work better on HX, airparks and BCP.

#### How can this be tested?
- pull down the brands branch
- pull down the tripapp branch https://github.com/holidayextras/tripapplite/pull/5396
- in the tripapp terminal `npm link ../brands/brands`
- go to the tripapp flow and change your url from `v2.holidayextras.co.uk:500` to `v2.parkbcp.co.uk:5000` and check the whole tripapp flow and make sure it is branded correctly and tripapp looks good. 
- check modals on `v2.holidayextras.co.uk:5000`, `v2.airparks.co.uk:5000` and `v2.parkbcp.co.uk:5000`

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![](http://tclhost.com/7PhcoIa.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
